nix/nixos-configurations/core-master/configuration.nix

```diff
@@ -27,15 +22,21 @@
     '';
   };
 
+  systemd.services.systemd-networkd-wait-online.enable = lib.mkForce false;
   # Make sure that the nix/machines/core/master.nixmakes of these files are actually lexicographically before 99-default.link provides by systemd defaults since first match wins
   # Ref: https://github.com/systemd/systemd/issues/9227#issuecomment-395500679
   networking.useDHCP = false;
   systemd.network = {
     enable = true;
+
+    netdevs.virbr0.netdevConfig = {
```

Comment:
idea here is to have physical interfaces on a bridge so that if we need to spin up VMs for AV, etc. we can just attach them directly to the bridge for network connectivity. We did pretty much the same thing with hypervisor* last year https://github.com/socallinuxexpo/scale-network/blob/e63ea79dffabafc03b0269b6d59788b301732fed/nix/nixos-configurations/hypervisor1/configuration.nix#L26-L62

Reply:
yup, agreed. this is good solution. I wonder if they'll attempt to setup the `caption` server this year.

Reply:
Yes theres nothing definitive but I think we should be ready to accommodate a VM or two.